Lib/test/test_aifc.py

```diff
@@ -149,6 +149,16 @@ def test_skipunknown(self):
         #This file contains chunk types aifc doesn't recognize.
         self.f = aifc.open(findfile('Sine-1000Hz-300ms.aif'))
 
+    def test_close_opened_files_on_error(self):
+        non_aifc_file = findfile('pluck-pcm8.wav', subdir='audiodata')
+        with check_no_resource_warning(self):
+            try:
+                # Try opening a non-AIFC file, with the expectation
+                # that `aifc.open` will fail.
+                f = self.f = aifc.open(non_aifc_file, 'rb')
+            except:
+                pass
```

Comment:
Could you use `with self.assertRaises(aifc.Error):` instead of `try` ~ `except: pass`?